pkg/ingress/apisix_upstream.go

```diff
@@ -122,6 +133,8 @@ func (c *apisixUpstreamController) sync(ctx context.Context, ev *types.Event) er
 	svc, err := c.controller.svcLister.Services(namespace).Get(name)
 	if err != nil {
 		log.Errorf("failed to get service %s: %s", key, err)
+		message := fmt.Sprintf(_messageResourceFailed, _upstreamController, err.Error())
```

Comment:
Can these two lines be encapsulated into one function? they used in many places, and once you need to change, you have to change everywhere.

Reply:
Yes, I will refactor it.